### ISSUE TITLE
Error on update to 3.0.1

### ISSUE THREAD

**Issue by migolo** [MoOx/postcss-cssnext#404]
The added script "postinstall" on package.json breaks the installation of the package on npm install command.
```
 "postinstall": "npm run babelify",
```

Can we remove it?

I am using Node v7.10.0 and npm v4.2.0

**Comment by migolo** [comment]
Error and NPM trace is:
```
sh: babel: command not found

npm ERR! Darwin 16.6.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "babelify"
npm ERR! node v7.10.0
npm ERR! npm  v4.2.0
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! postcss-cssnext@3.0.1 babelify: `babel src --out-dir lib`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the postcss-cssnext@3.0.1 babelify script 'babel src --out-dir lib'.
```

**Comment by passcod** [comment]
Also getting a build error on 3.0.1: https://travis-ci.org/passcod/mechanicalbird/builds/251869693?utm_source=github_status&utm_medium=notification

**Comment by MoOx** [comment]
Oooopsy! Fixed in 3.0.2 😊
